### PR TITLE
Remove Matomo from stack

### PR DESCRIPTION
**Pull request by birkland**
Remove Matomo module from composer and associated config; publish new
snapshot image with updated config, and without matomo content.

## To Test

Do `make up` and verify that the site still works.  

Note:  I cherry picked this onto the testcafe branch, and the simple "is the site still up" tests passed.

### CODE REVIEW

File: docker-compose.env.yml [hunk 1]
```diff
@@ -202,39 +202,5 @@ services:
         a3SNje4UXN+/1xUfQskxafP8CKVSr8xxtwzSureiskb5/98moAiutpUtp15yyAm0
         rwIDAQAB
         -----END PUBLIC KEY-----
-  matomo:
```
Comment (bseeger) [inline docker-compose.env.yml:227]: Git won't let me put a comment where I want to. On line 81 there is a reference to matomo in a Drupal URL that should probably go away.

**Review by bseeger — COMMENTED** [review]
Do you want to keep the docker-compose.matomo.yml file?  Not really an issue with it being there, but maybe it should go away? I don't have a strong opinion either way.

**Comment by bseeger** [comment]
Do you want to take matomo out of the .env file as well?

**Comment by bseeger** [comment]
Looks great, overall.   I ran this and confirmed that the matomo container doesn't exist and that Drupal doesn't have the module installed.

**Comment by birkland** [comment]
> 
> Do you want to keep the docker-compose.matomo.yml file? Not really an issue with it being there, but maybe it should go away? I don't have a strong opinion either way.

No, I think we should keep it in.  It's part of stock ISLE.  It'll make life more difficult to pull changes from upstream the more we touch the ISLE stuff.  As it stands, if it's not referenced then it's benign.

File: docker-compose.env.yml (hunk 1 above)
Comment (birkland) [reply docker-compose.env.yml:227]: Good catch.  I did cleanup of that, and the `.env` file, since they were easy to do, and pushed changes to this PR

**Review by bseeger — APPROVED** [review]
Changes look great!

**Pull request merged**